sdk/identity/Azure.Identity/src/DefaultAzureCredential.cs

```diff
@@ -56,7 +56,8 @@ internal DefaultAzureCredential(DefaultAzureCredentialFactory factory, DefaultAz
         }
 
         /// <summary>
-        /// Sequencially calls <see cref="TokenCredential.GetToken"/> on all the specified sources, returning the first successfully retured <see cref="AccessToken"/>.
+        /// Sequentially calls <see cref="TokenCredential.GetToken"/> on all the included credentials in the order <see cref="EnvironmentCredential"/>, <see cref="ManagedIdentityCredential"/>, <see cref="SharedTokenCacheCredential"/>,
+        /// and <see cref="InteractiveBrowserCredential"/> returning the first successfully obtained <see cref="AccessToken"/>. This method is called by Azure SDK clients. It isn't intended for use in application code.
```

Comment:
FWIW - `InteractiveBrowserCredential` is not called by default (unless `includeInteractiveCredentials` is set to `true` when constructing the `DefaultAzureCredential`, unsure if we want to call that out or not.